src/main/java/org/umssdiplo/automationv01/core/managepage/editItemType/EditItemType.java

```diff
@@ -0,0 +1,4 @@
+package org.umssdiplo.automationv01.core.managepage.editItemType;
+
+public class EditItemType {
```

Comment:
you need to use extends
after that it should have in the constructure some webelment that we need to wait.